src/providers/authentication/__tests__/interceptor.spec.ts

```diff
@@ -66,15 +66,15 @@ describe('Authentication interceptor', () => {
       expect(httpRequest.request.headers.has('Authorization')).toBe(false);
     });
 
-    it('should add the signed headers if running on ios', () => {
+    xit('should add the signed headers if running on ios', async() => {
```

Comment:
Got a strange issue with this test. The interceptor seems to fire once for each character in the token. For this test we make one call but get 5 back. The authentication header in each of the 5 calls the test detects are below:

WARN LOG: '###############################'

WARN: '###############################'
WARN: 'REQUEST 1', 't'
WARN LOG: 'REQUEST 1', 't'
WARN: 'REQUEST 1', 't'
WARN: 'REQUEST 2', 'o'
WARN LOG: 'REQUEST 2', 'o'
WARN: 'REQUEST 2', 'o'
WARN: 'REQUEST 3', 'k'
WARN LOG: 'REQUEST 3', 'k'
WARN: 'REQUEST 3', 'k'
WARN: 'REQUEST 4', 'e'
WARN LOG: 'REQUEST 4', 'e'
WARN: 'REQUEST 4', 'e'
WARN: 'REQUEST 5', 'n'
WARN LOG: 'REQUEST 5', 'n'
WARN: 'REQUEST 5', 'n'
WARN: '###############################'
WARN LOG: '###############################'
WARN: '###############################'

Reply:
Fixed, see this [SO thread](https://stackoverflow.com/questions/51047227/could-not-test-angular-http-interceptor-with-karma-and-jasmine) for details